### PR TITLE
Corrected FDT, SDT metrics in case of empty outputs

### PR DESCRIPTION
**Pull request by ljaljushkin**

### CODE REVIEW

**Comment by ljaljushkin** [comment]
@eaidova

**Comment by eaidova** [comment]
@ljaljushkin could you please fix code style?

File: llm_bench/python/who_what_benchmark/whowhatbench/whowhat_metrics.py [hunk 1]
```diff
@@ -30,44 +30,61 @@ def evaluate_divergency(tokenizer, data_gold, data_prediction):
...
+
+            num_matched = sum(block.size for block in blocks)
+            sdt = (
+                len(b_indexes) - num_matched
+            )
+            sdt_list.append(sdt)
+            sdt_norm = sdt / len(b_indexes)
+            sdtn_list.append(sdt_norm)
+
+            if DEBUG:
```
[the remark this answers is not in the document]
Comment (ljaljushkin) [reply llm_bench/python/who_what_benchmark/whowhatbench/whowhat_metrics.py:78]: yes, this debug code is for matched pairs only.

**Comment by ljaljushkin** [comment]
> @ljaljushkin could you please fix code style?

done

**Review by eaidova — APPROVED** [review]

**Pull request merged**